hazelcast/src/main/java/com/hazelcast/internal/nio/tcp/TcpIpNetworkingService.java

```diff
@@ -59,18 +63,17 @@
     private final ILogger logger;
 
     private final Networking networking;
-    private final MetricsRegistry metricsRegistry;
     private final ServerSocketRegistry registry;
 
     private final ConcurrentMap<EndpointQualifier, EndpointManager<TcpIpConnection>> endpointManagers =
-            new ConcurrentHashMap<EndpointQualifier, EndpointManager<TcpIpConnection>>();
+            new ConcurrentHashMap<>();
     private final TcpIpUnifiedEndpointManager unifiedEndpointManager;
     private final AggregateEndpointManager aggregateEndpointManager;
 
     private final ScheduledExecutorService scheduler;
 
     // accessed only in synchronized block
-    private volatile TcpIpAcceptor acceptor;
+    private AtomicReference<TcpIpAcceptor> acceptorRef = new AtomicReference<>();
```

Comment:
can you make this field final

Reply:
Do we need volatile/AtomicReference here if the comment says "accessed only in synchronized block"? Adjust comment?